mbed_host_tests/host_tests_conn_proxy/conn_proxy.py

```diff
@@ -28,7 +28,7 @@
 class KiViBufferWalker():
     """! Simple auxiliary class used to walk through a buffer and search for KV tokens """
     def __init__(self):
-        self.KIVI_REGEX = r"\{\{([\w\d_-]+);([^\}]+)\}\}\n"
```

Comment:
Since the key value protocol is based on start ```{{``` and end ```}}`` tags. I would suggest removing ```\n```. That way it will be more robust to match. Even if on some platform ```\n``` goes missing.